sriov/sriov.go

```diff
@@ -29,6 +29,7 @@ func setupVF(conf *SriovConf, ifName string, netns ns.NetNS) error {
 	var (
 		err       error
 		vfDevName string
+		locker *FileLocker
```

Comment:
How about make `locker` as global variable and initialize it in `init` function?